tests/test_db.py

```diff
@@ -203,6 +209,59 @@ def test_organization_not_found(self):
             db.find_organization('Bitergia')
 
 
+class TestFindGroup(TestCase):
+    """Unit tests for find_group"""
+
+    def setUp(self) -> None:
+        self.org = Organization.objects.create(name='Example')
+
+    def test_find_group(self):
+        """Check if a group is found by its name and organization"""
+
+        Group.add_root(name='Example Subgroup', organization=self.org)
+        group = db.find_group(organization=self.org, group_name='Example Subgroup')
+
+        self.assertIsInstance(group, Group)
+        self.assertEqual(group.organization.name, 'Example')
+        self.assertEqual(group.name, 'Example Subgroup')
+        self.assertEqual(group.get_parent(), None)
+
+    def test_group_not_found(self):
+        """Check if an error is raise if group is not found"""
+        with self.assertRaisesRegex(NotFoundError, GROUP_NOT_FOUND_ERROR):
+            db.find_group(self.org, 'subGroup')
```

Comment:
Leave a blank line between the docstring and the code.

Reply:
Done 👍